src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -165,20 +160,28 @@ start_metadata_sync_to_node(PG_FUNCTION_ARGS)
 	text *nodeName = PG_GETARG_TEXT_P(0);
 	int32 nodePort = PG_GETARG_INT32(1);
 
+	EnsureSuperUser();
```

Comment:
Should we throw error on `start_metadata_sync_to_node ` when `citus.enable_metadata_sync_by_default = false`?

Because otherwise the experience is pretty confusing, the user explicitly sync the metadata, but we don't because  `enable_metadata_sync_by_default == false`. It seems better that user knows that metadata is not synced?